### ISSUE TITLE
Empty constant value doesn't get updated

### ISSUE THREAD

**Issue by ghost** [pajtai/grunt-php-set-constant#5]
For example

``` php
define("CACHE_BUSTING_REVISION", "");
```

Task used 

``` javascript
grunt.initConfig({
    setPHPConstant: {
        stage: {
            constant: 'CACHE_BUSTING_REVISION',
            value: '<%= grunt.template.today("yyyymmddhh") %>',
            file: 'config/application.inc.php'
        }
    }
});

grunt.loadNpmTasks('grunt-php-set-constant');

grunt.registerTask('cachebust', 'setPHPConstant');
```

Line isn't updated when running grunt task.

It works when I set the initial value to 0 tho, so I can deal with that, but would be great if it would work with an empty string!

Thanks for the work by the way, extremely useful!

**Comment by pajtai** [comment]
Thanks for pointing this out falzhobel. It should be fixed in 0.0.12.

**Comment by ghost** [comment]
My pleasure!

Edit: Confirmed, works like a charm!
